content/github/administering-a-repository/displaying-a-sponsor-button-in-your-repository.md

```diff
@@ -16,7 +16,7 @@ You can add one username, package name, or project name per external funding pla
 
 Platform | Syntax
 -------- | -----
-[CommunityBridge](https://communitybridge.org) | `community_bridge: PROJECT-NAME`
+[LFX Mentorship (formerly CommunityBridge)](https://lfx.linuxfoundation.org/tools/mentorship) | `lfx_mentorship: PROJECT-NAME`
```

Comment:
```suggestion
[LFX Mentorship (formerly CommunityBridge)](https://lfx.linuxfoundation.org/tools/mentorship) | `community_bridge: PROJECT-NAME`
```

The syntax for this badge is remaining the same, so we won't need to change it in this PR.